src/transformers/training_args.py

```diff
@@ -532,6 +532,9 @@ class TrainingArguments:
             If True, the data loader will not shut down the worker processes after a dataset has been consumed once.
             This allows to maintain the workers Dataset instances alive. Can potentially speed up training, but will
             increase RAM usage. Will default to `False`.
+        dataloader_prefetch_factor (`int`, *optional*, defaults to None):
```

Comment:
small nit - we don't specify the value if the default is None in the docstrings

```suggestion
        dataloader_prefetch_factor (`int`, *optional*):
```